contracts/IICO.sol

```diff
@@ -249,7 +250,7 @@ contract IICO {
         if (msg.value != 0 && now >= startTime && now < endTime) // Make a bid with an infinite maxVal if some ETH was sent.
             submitBid(INFINITY, TAIL);
         else if (msg.value == 0 && finalized)                    // Else, redeem all the non redeemed bids if no ETH was sent.
-            for (uint i = 0; i < contributorBidIDs[msg.sender].length; ++i)
+            for (uint i = 0; i < contributorBidIDs[msg.sender].length; ++i) //WARNING: High gas consumption for high length, possible transaction fail, no gas revert
```

Comment:
Yes, a user can on purpose make his own fallback revert. But he will then still be able to redeem individually.

Reply:
Only a disclaimer comment to the users that actually don't intend to make bad contract usage since there will be no gas revert